### PR TITLE
Change the color of (Owner) text

### PR DESCRIPTION
**Pull request by BrownHedgehog78**

### CODE REVIEW

File: pvp_club/init.lua [hunk 1]
```diff
@@ -78,7 +78,8 @@ minetest.register_on_joinplayer(function(player, n)
                     text = member
                 }
                 if is_owner then
-                    props.text = props.text.." (Owner)"
+                    props.text = props.text..
+                   mt.colorize("#da1818", " (Owner)")
```
Comment (Anthony-De) [inline pvp_club/init.lua]: The line isn't too long so it doesn't need to be on 2 lines.

**Review by Anthony-De — APPROVED** [review]

File: pvp_club/init.lua (hunk 1 above)
Comment (BrownHedgehog78) [reply pvp_club/init.lua]: Huh?

Comment (Anthony-De) [reply pvp_club/init.lua]: You can use it like this:
```lua
props.text = props.text..mt.colorize("#da1818", " (Owner)")
```
Using 2 lines is unneeded.

Comment (BrownHedgehog78) [reply pvp_club/init.lua]: Ok

**Comment by BrownHedgehog78** [comment]
Will be merged?

**Pull request merged**